week2/tests.py

```diff
@@ -23,12 +20,6 @@ def ex2runs():
         return False
 
 
-def strip_docstring(a_function):
-    regex = r"\"\"\"[\s\w\.\'\"\&\!\#\%]*\"\"\""
```

Comment:
It seems that this function doesn't strip the docstring properly. Thus we get false positves. I.e tests pass if output is correct and even though the source does not implement a map